subcommands/waves/init.go

```diff
@@ -54,6 +61,20 @@ Example: 1,2,3`)
 	_ = initCmd.MarkFlagRequired("keys")
 }
 
+func confirmWaveInitProceed() {
+	logrus.Warning("**(Recommended)** Generate OSTree static deltas before rolling out waves to devices. " +
+		"More info: https://docs.foundries.io/latest/reference-manual/ota/static-deltas.html#ref-static-deltas \n")
+
+	logrus.Infof("Do you want to proceed with the wave init? (y/n): ")
```

Comment:
I think we can be smarter than this.

Based on [this code](https://github.com/foundriesio/ci-scripts/blob/master/lmp/static_deltas.py#L90) I see that we can easily identify if a given target has static deltas or not.
So, it only makes sense to warn the user in case if there are no static deltas, and silently proceed if everything is fine.

Reply:
I think this is what I and @doanac agreed on in the slack channel thread.

I think it's better to make it a non-interactive too.

I don't have a strong opinion whether to print the warning message in the beginning or at the end of the command output.